docs/identity/authentication/howto-sspr-windows.md

```diff
@@ -12,120 +12,125 @@ author: justinha
-    - *Explorer.exe* is replaced with a custom shell.
-    - **Interactive logon: Require smart card** is set to **Enabled** or **1**.
+    - *HideFastUserSwitching* is set to enabled or 1
+    - *DontDisplayLastUserName* is set to enabled or 1
+    - *NoLockScreen* is set to enabled or 1
+    - *BlockNonAdminUserInstall* is set to enabled or 1
+    - *EnableLostMode* is set on the device
+    - Explorer.exe is replaced with a custom shell
+    - Interactive logon: Require smart card is set to enabled or 1
 - The combination of the following specific three settings can cause this feature to not work.
-    - **Interactive logon: Do not require CTRL+ALT+DEL** is set to **Disabled** (only for Windows 10 version 1710 and earlier).
-    - `DisableLockScreenAppNotifications` is set to **Enabled** or **1**.
-    - Windows version is the Home edition.
+    - Interactive logon: Do not require CTRL+ALT+DEL = Disabled (only for Windows 10 version 1710 and earlier)
+    - *DisableLockScreenAppNotifications* = 1 or Enabled
+    - Windows SKU is Home edition
 
 > [!NOTE]
 > These limitations also apply to Windows Hello for Business PIN reset from the device lock screen.
 >
-
+## Proposed additional information ##
+**Clarification Needed on Looping Behavior in Windows 11 During Transition to Default User Session**
+**SSPR Compatibility with Windows 11:** Unlike previous versions such as Windows 10 22H2, Windows 11 does not support initiating the SSPR process from a locked user session. The SSPR must be initiated from the logon screen, ensuring that sessions currently locked out do not complicate or interfere with the reset process.
+**SSPR Process Overview:** To securely initiate the process, the system requires an appropriate session state. To achieve this, the system transitions out of the secure lock/unlock session by utilizing the session defined under "DefaultUser",  which is internally generated by the SSPR service. This intermediary step is crucial as it enables the launch of the Web Dialog for SSPR, ensuring the process adheres to security protocols.
+**Technical Complexity and Misunderstandings:** The SSPR process involves multiple system components that must synchronize within specific time constraints, such as a 5-second window for RPC calls. There has been some confusion regarding the responsibilities of components, especially concerning Entra ID Authentication and LogonUI parts, which are sometimes perceived as separate from the SSPR process.
+**Performance and Error Handling:** If the transition to the secure session does not occur within the expected timeframe, the system defaults to the logon screen. This behavior, which may resemble a service repeatedly looping on the logon screen, can be mistakenly interpreted as an error. However, the system's response is deliberately designed to prevent unauthorized access attempts. This security-oriented functionality, while essential, might not align with typical user expectations. 
```

Comment:
```suggestion
**Performance and Error Handling:** If the transition to the secure session does not occur within the expected timeframe, the system defaults to the logon screen. This behavior, which may resemble a service repeatedly looping on the logon screen, can be mistakenly interpreted as an error. However, the system's response is deliberately designed to prevent unauthorized access attempts. This security-oriented functionality, while essential, might not align with typical user expectations. 

```